rust/protocol/src/protocol.rs

```diff
@@ -15,7 +22,10 @@ use sha2::Sha256;
 use subtle::ConstantTimeEq;
 use uuid::Uuid;
 
+/// Each type of message in the Signal Protocol has a message version mixed into it when
+/// created. Prefer to reference this value over use of the literal version number.
 pub const CIPHERTEXT_MESSAGE_CURRENT_VERSION: u8 = 3;
+/// Each [SenderKeyRecord] has a message version field as well.
 pub const SENDERKEY_MESSAGE_CURRENT_VERSION: u8 = 3;
```

Comment:
These comments should each stand alone, though you could add a "see also". If we're adding doc comments, they should look good in the generated rustdoc output as well as in the source, and that might not put them next to each other.